dockers/docker-pde/base_image_files/port_breakout.py

```diff
@@ -0,0 +1,833 @@
+#! /usr/bin/python -u
+
+import os
+import sys
+import re
+import getopt
+import copy
+import json
+import shutil
+import os.path
+import subprocess
+
+SIM_HOST = False
+
+SAI_PROFILE_DELIMITER = '='
+INTERFACE_KEY="Ethernet"
```

Comment:
Add space on each side of `=`

Please pass all Python code through http://pep8online.com/ and fix reported non-compliance. However, lines over 80 chars are OK. Start to consider breaking lines around 100 chars.

Reply:
This file was removed from the PR....its no longer in use/needed.